celery/schedules.py

```diff
@@ -408,6 +408,14 @@ def __init__(self, minute: str = '*', hour: str = '*', day_of_week: str = '*',
         self.month_of_year = self._expand_cronspec(month_of_year, 12, 1)
         super().__init__(**kwargs)
 
+    @classmethod
+    def from_string(cls, crontab: str):
+        """
+        Create a Crontab from a string. For example ``crontab='* * * * *'``.
+        """
+        minute, hour, day_of_week, day_of_month, month_of_year = crontab.split(" ")
```

Comment:
I would suggest to switch the order of fields to match offical cron format
```suggestion
        minute, hour, day_of_month, month_of_year, day_of_week = crontab.split(" ")
```

```
# ┌───────────── minute (0–59)
# │ ┌───────────── hour (0–23)
# │ │ ┌───────────── day of the month (1–31)
# │ │ │ ┌───────────── month (1–12)
# │ │ │ │ ┌───────────── day of the week (0–6) (Sunday to Saturday;
# │ │ │ │ │                                   7 is also Sunday on some systems)
# │ │ │ │ │
# │ │ │ │ │
# * * * * * <command to execute>
```
Source: https://en.wikipedia.org/wiki/Cron